packages/manager/modules/nutanix/src/dashboard/nodes/node/general-info/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,17 @@
+{
+  "server_datacenter_ERI": "London (ERI {{number}}) - England",
```

Comment:
Maybe the `Region` service could be reused for this purpose (https://github.com/ovh/manager/tree/develop/packages/manager/modules/manager-components/src/region)

Reply:
This has to be done in `bm-server-components` and not here. As this is used in one of the components used here.
Also this means, `bm-server-components` to be rebased with latest `develop` branch as well